tests/components/nsw_rural_fire_service_feed/test_geo_location.py

```diff
@@ -110,12 +114,12 @@ def _generate_mock_feed_entry(
     mock_entry_3 = _generate_mock_feed_entry("3456", "Title 3", 25.5, (-31.2, 150.2))
     mock_entry_4 = _generate_mock_feed_entry("4567", "Title 4", 12.5, (-31.3, 150.3))
 
-    utcnow = dt_util.utcnow()
     # Patching 'utcnow' to gain more control over the timed update.
+    utcnow = dt_util.utcnow()
     with patch("homeassistant.util.dt.utcnow", return_value=utcnow), patch(
-        "geojson_client.nsw_rural_fire_service_feed." "NswRuralFireServiceFeed"
-    ) as mock_feed:
-        mock_feed.return_value.update.return_value = (
+        "aio_geojson_client.feed.GeoJsonFeed.update", new_callable=CoroutineMock
```

Comment:
Isn't `CoroutineMock` the default new callable for coroutines patched with `asynctest.patch`?

Reply:
According to the document you are right:

> By default, the target will be patched with an instance of CoroutineMock if it is a coroutine, or a MagicMock object.

Let me test this tonight.

Reply:
I removed the two occurrences of `new_callable=CoroutineMock`. Tests are working as expected without this.